.ci/generate_bear_requirements.py

```diff
@@ -155,18 +206,32 @@ def get_pip_requirements(requirements):
     pip_reqs, npm_reqs, gem_reqs = (
         get_all_requirements(get_all_bears(bear_dirs)))
 
-    requirements = {}
-    requirements['overrides'] = 'coala-build.yaml'
-    requirements['pip_requirements'] = get_pip_requirements(pip_reqs)
-    requirements['npm_requirements'] = get_npm_requirements(npm_reqs)
+    requirements = CommentedMap()
     requirements['gem_requirements'] = get_gem_requirements(gem_reqs)
+    requirements['npm_requirements'] = get_npm_requirements(npm_reqs)
+    requirements['pip_requirements'] = get_pip_requirements(pip_reqs)
+
+    if args.update or args.check:
+        input_file = open(os.path.join(PROJECT_DIR, BEAR_REQUIREMENTS_YAML),'r')
```

Comment:
if the file doesnt exist, we dont want a backtrace.

also you have a pycodestyle error here.